### PR TITLE
feat(MA-6): add additional fields to events

### PR DESCRIPTION
**Pull request by HassanCehef**
EQAC requested five new fields to be added when events are created.

This must be merged in conjunction with the corresponding change in frontend

### CODE REVIEW

File: models/Event.js [hunk 1]
```diff
@@ -334,7 +343,34 @@ const Event = sequelize.define('event', {
         type: Sequelize.TEXT,
         allowNull: true
     },
-}, {
+    optional_programme: {
+        type: Sequelize.TEXT,
+        allowNull: true
+    },
+    link_info_travel_country: { // not validated. Will be checked client side, but validated nowhere
```
Comment (serge1peshcoff) [inline models/Event.js:350]: not sure what does this field means, what it'll be used for?

Comment (WikiRik) [reply models/Event.js:350]: From the ticket; "Please insert a link which provides useful information about travelling to your country! (optional open question, string)"

File: test/scripts/generator.js [hunk 2]
```diff
@@ -32,6 +33,10 @@ exports.generateEvent = (options = {}) => {
     if (notSet(options.budget)) options.budget = faker.lorem.sentence();
     if (notSet(options.programme)) options.programme = faker.lorem.sentence();
 
+    if (notSet(options.accommodation_type)) options.accommodation_type = faker.lorem.sentence();
+    if (notSet(options.optional_programme)) options.optional_programme = faker.lorem.sentence();
+    if (notSet(options.meals_per_day)) options.meals_per_day = faker.random.number({ min: 5, max: 3 });
```
Comment (WikiRik) [inline test/scripts/generator.js:38]: Minimum of 5, maximum of 3?

File: .editorconfig [hunk 3]
```diff
@@ -16,7 +16,7 @@ charset = utf-8
 # 4 space indentation
 [*.js]
 indent_style = space
-indent_size = 2
+indent_size = 4
```
[the remark this answers is not in the document]
Comment (WikiRik) [reply .editorconfig:19]: @HassanCehef can you check this out?

File: test/scripts/generator.js (hunk 2 above)
Comment (HassanCehef) [inline test/scripts/generator.js:38]:
```suggestion
    if (notSet(options.meals_per_day)) options.meals_per_day = faker.random.number({ min: 0, max: 3 });
```

File: test/scripts/generator.js [hunk 4]
```diff
@@ -14,6 +14,7 @@ exports.generateEvent = (options = {}) => {
     if (notSet(options.ends)) options.ends = faker.date.future(null, options.starts);
     if (notSet(options.type)) options.type = faker.random.arrayElement(['training', 'nwm', 'conference', 'cultural']);
     if (notSet(options.fee)) options.fee = faker.random.number({ min: 0, max: 100 });
+    if (notSet(options.optional_fee)) options.optional_fee = faker.random.number({ min: 0, max: 2 });
```
Comment (HassanCehef) [inline test/scripts/generator.js:17]:
```suggestion
    if (notSet(options.optional_fee)) options.optional_fee = faker.random.number({ min: 0, max: 20 });
```

File: .editorconfig (hunk 3 above)
[the remark this answers is not in the document]
Comment (HassanCehef) [reply .editorconfig:19]: The value was different from the comment line 16. I unified it. @serge1peshcoff can comment.

[the remark this answers is not in the document]
Comment (WikiRik) [reply .editorconfig:19]: 4 is indeed conform the linter

[the remark this answers is not in the document]
Comment (serge1peshcoff) [reply .editorconfig:19]: yep, eslint is using 4 spaces

**Review by WikiRik — APPROVED** [review]

**Pull request merged**